cypress/integration/BounceRulesContainer/bounce_rules.spec.js

```diff
@@ -14,4 +14,25 @@ describe("Bounce Rules Page", () => {
     BounceRulesPage.ruleFilter.should("be.visible");
     BounceRulesPage.ruleTable.should("be.visible");
   });
+
+  it("should create a bounce rule", () => {
+    BounceRulesPage.createRuleButton.click();
+    BounceRulesPage.fillForm(
+      "1",
+      "cypressTest",
+      "528",
+      "testDescription",
+      "492",
+      "testRegex"
+    );
+    BounceRulesPage.confirmModal.should("be.visible");
+    BounceRulesPage.confirmationSubmit.click();
+    BounceRulesPage.testRule.should("be.visible");
+  });
+
+  it("should delete a bounce rule", () => {
+    BounceRulesPage.testRule.click();
```

Comment:
On another note as well, let's have two separate bounce rules: one that we will use in the create bounce rule test and another in the delete bounce rule test. Imagine if we were running the tests in parallel or someone else runs it at the same time as you hitting the same testing database. We may stomp on each other's resources if we're using the same test rule. It's often best to have separate resources in this case to avoid as much collisions as possible